PySoar/exportClass.py

```diff
@@ -51,7 +51,7 @@ def initiate_best_worst(self, settings, no_legs):
     def __init__(self, settings, no_legs):
         self.file_name = settings.file_name
 
-        self.wb = xlwt.Workbook()  # initialize excel sheet
+        self.wb = xlwt.Workbook(encoding='latin-1')  # initialize excel sheet
```

Comment:
I added this because PySoar crashes during the generation of an excel sheet in case there is an umlaut (ä,ö,ü) in the competition id. Try the following day that did not work for me:
http://www.soaringspot.com/en_gb/qualifikationsmeisterschaft-landau-landau-ebenberg-2016/results/15m-klasse/task-5-on-2016-08-27/daily